src/Sarif.Sarifer/SariferPackage.cs

```diff
@@ -73,6 +74,11 @@ await this.GetServiceAsync(typeof(SVsShell)).ConfigureAwait(continueOnCapturedCo
                 this.analyzeProjectCommand = new AnalyzeProjectCommand(mcs);
             }
 
+            if (await GetServiceAsync(typeof(SVsOutputWindow)).ConfigureAwait(continueOnCapturedContext: true) is IVsOutputWindow output)
+            {
+                outputWindowTraceListener = new OutputWindowTraceListener(output, "Sarifer");
```

Comment:
>outputWindowTraceListener = new OutputWindowTraceListener(output, "Sarifer"); [](start = 16, length = 77)

with this, when we execute and use Trace.Write / Trace.WriteLine, we would see a new dropdown in the output windows with the data that we trace. Here I do not know if we should ship on Release mode. What do you think?

Reply:
Fine for now! This whole package is experimental.

---
In reply to: [550914135](https://github.com/microsoft/sarif-visualstudio-extension/pull/338#discussion_r550914135) [](ancestors = 550914135)